katalog/dual-nginx/kustomization.yaml

```diff
@@ -13,4 +12,4 @@ resources:
 
 imageTags:
```

Comment:
please remove the imageTags block, deployments should use the currect tag directly